docs/playbook.md

```diff
@@ -18,6 +18,11 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
 ### Mount successful but files not visible
 This is explained in the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories) page and can possibly be fixed by passing the --implicit-dirs=true flag to the gcsfuse command.
 
+### Mount failed with fusermount3 exit status 1
+
```

Comment:
there is no new line between header and description for others. please remove it here.

Reply:
fixed